### PR TITLE
Flags

### PR DESCRIPTION
**Pull request by retabak**

### CODE REVIEW

File: lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc
Comment (schanzer) [inline lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc:1]: This workbook page has a bunch of display glitches, due to messed-up brackets. I'm pushing a commit to fix them. Reminder to open the pages and visually inspect before pushing.

File: lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+For this section of the page, you will refer to the @starter-file{flags}.
+
+@n @ifproglang{wescheme}{Each language has its own symbol for commenting. In WeScheme, we use the semi colon (`;`) to tell the computer to ignore what we're typing. What color does the code turn when it has a semicolon in front of it?} @ifproglang{pyret}{In Pyret, we use the number sign (`#`) to tell the computer to ignore what we're typing. What color does the code turn when it has a number sign in front of it?}@fitb{25px}{@ifsoln{@ifproglang{wescheme}{Purple.}@ifproglang{pyret}{Red.}}}
+
+
+@n Type `japan-flag` into the Interactions Area. What do you get back?@fitb{}{@ifsoln{We see Japan's flag - a white background with}
+
+@fitb{}{a red dot in the center.}
+
+@n Type `japan` into the Interactions Area and compare the image to `japan-flag`. How are they alike? How are they different?@fitb{}{}
```
Comment (schanzer) [inline lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc]: Just curious: why do you not have a space before `@fitb{}{}`? It seems like you are consciously removing them, but I don't know why.

File: lessons/Projects/project-flags/langs/en-us/slides.md
Comment (schanzer) [inline lessons/Projects/project-flags/langs/en-us/slides.md:1]: What is this file doing here, and why is it all about building videogames?

File: shared/langs/en-us/starter-files.js
Comment (schanzer) [inline shared/langs/en-us/starter-files.js:1]: Why is this file part of the PR?

**Review by schanzer — CHANGES_REQUESTED** [review]
I committed a few changes - please let me know if you're ok with them. Overall I think this is really good! But the PR seems to be polluted by another branch, which has me worried. I think it's worth doing a branch cleanup here before merging.

File: lessons/Algebra/flags/langs/en-us/index.adoc [hunk 2]
```diff
@@ -69,147 +69,76 @@ Think of the background image as a sheet of graph paper with the @vocab{origin}
...
 Professional programmers use comments __all the time__.
 }
 
 Professional programmers work with teams who need to be able to follow each other's thinking in order to collaborate efficiently. Comments are a way for programmers to leave notes for one another, and even for a single programmer to keep track of their thinking when _they_ come back to their code another day.
 
-@lesson-instruction{
-- @ifproglang{wescheme}{Each language has its own symbol for commenting. In WeScheme, we use the semi colon (`;`) to tell the computer to ignore what we're typing. What color does the code turn when it has a semicolon in front of it?
-** _Purple!_}
-@ifproglang{pyret}{In Pyret, we use the number sign (`#`) to tell the computer to ignore what we're typing. What color does the code turn when it has a number sign in front of it?
-** _Red!_}
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Algebra/flags/langs/en-us/index.adoc:97]: I didn't write this! But I'll change it to "orange-ish red"? how do you describe that color?

File: lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc
Comment (retabak) [reply lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc:1]: thank you.

File: lessons/Algebra/flags/langs/en-us/index.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Algebra/flags/langs/en-us/index.adoc:97]: oh! Sorry - it just caught my eye. I'd call it "dark orange"

File: lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc (hunk 1 above)
Comment (retabak) [reply lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc]: Lol. I think I wanted to make sure the @fitb line started immediately. Clearly this was silly of me. I can stop this!

File: lessons/Projects/project-flags/langs/en-us/slides.md
Comment (retabak) [reply lessons/Projects/project-flags/langs/en-us/slides.md:1]: woopsy.

**Pull request merged**